devfiles/java-springboot/devfile.yaml

```diff
@@ -9,44 +9,34 @@ starterProjects:
 components:
   - container:
       name: tools
-      image: maysunfaisal/springbootbuild
+      image: quay.io/eclipse/che-java11-maven:nightly
       memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       mountSources: true
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - container:
-      name: runtime
-      image: maysunfaisal/springbootruntime
-      memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       endpoints:
-        - name: '8080/tcp'
-          targetPort: 8080
-      mountSources: false
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - volume:
-      name: springbootpvc
-      size: 4Gi
+      - name: '8080-tcp'
+        targetPort: 8080
 commands:
   - exec:
       id: devBuild
```

Comment:
lets change this to `maven build` from devbuild

Reply:
I think lets rename it to simply `build`, during `odo push` it will look cleaner
` Executing build command "mvn package"`,